ui/package.json

```
@@ -56,14 +56,16 @@
       "node": true
     },
     "extends": [
-      "plugin:vue/essential",
       "eslint:recommended",
-      "@vue/prettier"
+      "plugin:vue/recommended",
+      "@vue/eslint-config-prettier"
     ],
     "parserOptions": {
       "parser": "babel-eslint"
     },
-    "rules": {}
+    "rules": {
+      "vue/multi-word-component-names": "off"
```

If you want to silence all the warnings, the other 2 rules can be turned off, but left them on, since they weren't error level messages
```suggestion
      "vue/multi-word-component-names": "off",
      "vue/require-prop-types": "off",
      "vue/require-default-prop": "off"
```